psdisk/psdisk.c

```diff
@@ -0,0 +1,220 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Phoenix Systems disk tool
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#include <stdio.h>
+#include <stdint.h>
+#include <unistd.h>
+#include <string.h>
+#include <ctype.h>
+#include <stdlib.h>
+#include <sys/list.h>
+
+/* TODO: change access to ptable.h */
+#include "../../phoenix-rtos-filesystems/ptable/ptable.h"
+
+
+#define LOG_ERROR(str, ...) do { fprintf(stderr, __FILE__  ":%d error: " str "\n", __LINE__, ##__VA_ARGS__); } while (0)
```

Comment:
User don't care about line of the code, this is a tool for debugging rather. Maybe something simpler for error reporting?